templates/aws/cluster-template.yaml

```diff
@@ -51,6 +51,13 @@ spec:
       name: ${CLUSTER_NAME}-control-plane
   spec:
     nodeName: "{{ ds.meta_data.local_hostname }}"
+    bootstrapConfig:
```

Comment:
this is probably to be removed before merging, right?

Reply:
Yes